programs/bench.h

```diff
@@ -13,12 +13,32 @@
 #define BENCH_H_121279284357
 
 #include <stddef.h>   /* size_t */
+#include "mem.h"
```

Comment:
The least amount of `#include` is always the better.
It's even more important inside a `.h` header file, because the compilation overhead is multiplied by the nb of users of the `.h`, which can quickly ramp up after a few nested levels.

In this case, I guess you need `mem.h` for the `U32` type within `BMK_benchMem()`.

Consider replacing this type by `unsigned` instead.